operator/es_client.go

```diff
@@ -388,9 +389,13 @@ func (c *ESClient) GetIndices() ([]ESIndex, error) {
 		return nil, err
 	}
 
-	// HACK: after-the-fact conversion of strings to integers from ES response.
 	returnStruct := make([]ESIndex, 0, len(esIndices))
 	for _, index := range esIndices {
+		// ignore system indices
+		if c.excludeSystemIndices && index.Index[0:1] == "." {
```

Comment:
Isn't this just `index.Index[0]` ? And is it always safe to do this, or should we check the length first?

Reply:
the length should always be > 0 as there's no chance of creating an index without name.

Reply:
I see, maybe a bit nicer way to do it is with: https://golang.org/pkg/strings/#HasPrefix then it's maybe more clear what we want to check and there's no doubt about the length or anything. This is just a nitpick.